src/views/tickets/orders-rsvp.php

```diff
@@ -77,10 +78,9 @@
 							$value = '&nbsp;';
 						}
 					if ( '' != $value ) { ?>
-						<a class="attendee-meta toggle show"><?php esc_html_e( 'Toggle attendee info', 'event-tickets-plus' ); ?></a>
 						<div class="attendee-meta-details">
-							<span class="event-tickets-meta-label <?php echo esc_attr( $field->slug ); ?>"><?php echo esc_html( $field->label ); ?>&nbsp;</span>
-							<span class="event-tickets-meta-data <?php echo esc_attr( $field->slug ); ?>"><?php echo $value ? esc_html( $value ) : '&nbsp;'; ?></span>
+							<span class="event-tickets-meta-label <?php echo esc_attr( $field->slug ); ?>"><?php echo esc_html( $field->label ); ?></span>
```

Comment:
I suspect this may need to be revised and moved to ET+ (see [this PR](https://github.com/moderntribe/event-tickets/pull/200/)).